### ISSUE TITLE
Please provide support for MySQL 5.6 and 5.7 on Debian

### ISSUE THREAD

**Issue by paweloczadly** [chef-cookbooks/mysql#291]
Hi Guys,

Could you implement support for MySQL 5.6 and 5.7 for Debian? Currently, when I use your cookbook I get:

* MySQL 5.6:
```
[2015-02-06T11:05:42+01:00] ERROR: Unsupported Version: You requested to install a Mysql Server version that is not supported by your platform       
[2015-02-06T11:05:42+01:00] ERROR: Platform: debian 7.8 - Request Mysql Server version: 5.6       
[2015-02-06T11:05:42+01:00] ERROR: Availabe versions for your platform are: 5.5       
       
           
    ================================================================================       
    Error executing action `create` on resource 'mysql_service[default]'       
    ================================================================================       
           
    RuntimeError       
    ------------       
    Unsupported Mysql Server Version       
           
    Cookbook Trace:       
    ---------------       
           /tmp/kitchen/cookbooks/mysql/libraries/helpers.rb:292:in `package_name_for'
           /tmp/kitchen/cookbooks/mysql/libraries/helpers.rb:328:in `server_package'
           /tmp/kitchen/cookbooks/mysql/libraries/helpers.rb:339:in `server_package_name'
           /tmp/kitchen/cookbooks/mysql/libraries/provider_mysql_service.rb:30:in `block in <class:MysqlService>'
```

* MySQL 5.7:
```
[2015-02-06T11:09:19+01:00] ERROR: Unsupported Version: You requested to install a Mysql Server version that is not supported by your platform       
[2015-02-06T11:09:19+01:00] ERROR: Platform: debian 7.8 - Request Mysql Server version: 5.7       
[2015-02-06T11:09:19+01:00] ERROR: Availabe versions for your platform are: 5.5       
       
           
    ================================================================================       
    Error executing action `create` on resource 'mysql_service[default]'       
    ================================================================================       
           
    RuntimeError       
    ------------       
    Unsupported Mysql Server Version       
           
    Cookbook Trace:       
    ---------------       
    /tmp/kitchen/cookbooks/mysql/libraries/helpers.rb:292:in `package_name_for'       
    /tmp/kitchen/cookbooks/mysql/libraries/helpers.rb:328:in `server_package'       
    /tmp/kitchen/cookbooks/mysql/libraries/helpers.rb:339:in `server_package_name'
    /tmp/kitchen/cookbooks/mysql/libraries/provider_mysql_service.rb:30:in `block in <class:MysqlService>'
```

**Comment by someara** [comment]
What's the canonical way to get these packages on the debian family?

**Comment by shaneramey** [comment]
You should specify a different version, because 5.5 is the only version available
```
mysql_service 'default' do
  version '5.5'
  bind_address '0.0.0.0'
  port '3306'  
  data_dir '/data'
  initial_root_password dbpassword
  action [:create, :start]
end
```

**Comment by arosenhagen** [comment]
This gets really annoying every once in a while. I hope there will be a version of this cookbook which doesn't hardcode versions in such a strict way in the future. We also use the apt-repo from dotdeb, where mysql-5.6 and mysql-5.7 is available for debian7 and we always have to keep a fork (https://github.com/ffuenf/mysql/commit/3ebcfef3ebbec45bf05e7ebd12e9294f3187e4ff) to allow successful installation of the mysql-service.

**Comment by someara** [comment]
Released in v6.0.18
